### PR TITLE
feat: feat upload new icon

### PR DESCRIPTION
**Pull request by wangxueliang**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

File: components/upload/UploadList.jsx [hunk 1]
```diff
@@ -3,7 +3,10 @@ import { getOptionProps, initDefaultProps, getListeners } from '../_util/props-u
 import getTransitionProps from '../_util/getTransitionProps';
 import { ConfigConsumerProps } from '../config-provider';
 import { previewImage, isImageUrl } from './utils';
-import Icon from '../icon';
+import {
+  UploadOutlined, PaperClipOutlined, PictureOutlined, FileOutlined,
+  DeleteOutlined, DownloadOutlined, EyeOutlined,
+} from '@ant-design/icons-vue';
```
Comment (Amour1688) [inline components/upload/UploadList.jsx]: 直接写到路径，不然还是全量引入了

File: components/upload/UploadList.jsx [hunk 2]
```diff
@@ -93,13 +96,13 @@ export default {
...
       let progress;
-      let icon = <Icon type={file.status === 'uploading' ? 'loading' : 'paper-clip'} />;
+      let icon = file.status === 'uploading' ? <UploadOutlined /> : <PaperClipOutlined />;
 
       if (listType === 'picture' || listType === 'picture-card') {
         if (listType === 'picture-card' && file.status === 'uploading') {
           icon = <div class={`${prefixCls}-list-item-uploading-text`}>{locale.uploading}</div>;
         } else if (!file.thumbUrl && !file.url) {
-          icon = <Icon class={`${prefixCls}-list-item-thumbnail`} type="picture" theme="twoTone" />;
+          icon = <PictureOutlined class={`${prefixCls}-list-item-thumbnail`} theme="twoTone" />;
```
Comment (tangjinzhou) [inline components/upload/UploadList.jsx]: 没有theme了  <PictureTwoTone />

File: components/upload/UploadList.jsx [hunk 3]
```diff
@@ -108,7 +111,7 @@ export default {
               class={`${prefixCls}-list-item-image`}
             />
           ) : (
-            <Icon type="file" class={`${prefixCls}-list-item-icon`} theme="twoTone" />
+            <FileOutlined class={`${prefixCls}-list-item-icon`} theme="twoTone" />
```
Comment (tangjinzhou) [inline components/upload/UploadList.jsx]: 同上

File: .gitignore [hunk 4]
```diff
@@ -61,6 +61,7 @@ typings/
 .DS_Store
 dist
 lib
+antdv-demo
```
Comment (tangjinzhou) [inline .gitignore]: 这个要保留啊

**Pull request merged**